astropy/visualization/interval.py

```diff
@@ -71,15 +76,16 @@ def __call__(self, values, clip=True, out=None):
             values = np.subtract(values, float(vmin))
         else:
             if out.dtype.kind != 'f':
-                raise TypeError('Can only do in-place scaling for '
-                                'floating-point arrays')
+                raise TypeError(
+                    'Can only do in-place scaling for ' 'floating-point arrays'
```

Comment:
Not that I know of, but I haven't really tried that hard to find an automatic fix.

Reply:
Yes `--preview` activates some experimental string processing but it's still experimental:
https://black.readthedocs.io/en/stable/the_black_code_style/future_style.html#improved-string-processing
It seems a bit risky to use it as the additional changes caused by this option would be lost in the middle of the other changes, maybe we can give it a try after the black PRs have been merged ?

Reply:
It is possible to review changes one commit at a time, so applying `black --preview` in a separate commit after `black` has already been applied should be manageable. And this will not clutter the history because we'll squash every PR down to a single commit anyways.

Reply:
Sure it is possible but will people do it ? I think it's a bit late to introduce an experimental option. I use this option in small projects but here given the number of changes it would be easy to miss some weird behavior.

Reply:
I wouldn't want to merge this as is.  So should maintainers manually fix these string concatenations?

Reply:
I'll try go through and fix these concatenations and push a commit.